test/e2e/support/utils/disable-pre-publish-checks.js

```diff
@@ -3,6 +3,9 @@
  */
 import { toggleOption } from './toggle-option';
 
+/**
+ * Disable Pre-publish checks
```

Comment:
Can we use the form of 3rd person for consistency and end the sentence with a full stop?

Reply:
Sure, thx.